src/test/ledger/Invariants_test.cpp

```diff
@@ -55,6 +55,7 @@ class Invariants_test : public beast::unit_test::suite
     // changes that will cause the check to fail.
     void
     doInvariantCheck( bool enabled,
+        std::vector<std::string> const& expect_logs,
```

Comment:
This seems like a new feature of doInvariantCheck() that's not reflected in the commit message. Namely, the additional parameter appears to be used to confirm that the test fails as returning the expected bad circumstance.